packages/framework/src/Model/Customer/User.php

```diff
@@ -181,14 +181,6 @@ public function setResetPasswordHash($hash)
         $this->resetPasswordHashValidThrough = new DateTime('+48 hours');
     }
 
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Customer\DeliveryAddress|null $deliveryAddress
-     */
-    public function setDeliveryAddress(DeliveryAddress $deliveryAddress = null)
```

Comment:
I love what you are doing in this PR :+1: :slightly_smiling_face:

Reply:
we love the work too